### ISSUE TITLE
Adding a quickcheck interoperability layer

### ISSUE THREAD

**Issue by Centril** [AltSysrq/proptest#18]
I've created an interoperability layer between proptest and quickcheck, should I add this to proptest as a PR under a feature flag `quickcheck` or do you believe this belongs in a separate crate?

The benefit of the latter is that proptest is not then constrained in dependencies by `quickcheck` while the benefit of the former is that the user doesn't have to write in another dependency to their project.

What it amounts to is essentially (details omitted):

```rust
use quickcheck::Arbitrary;

pub fn from_qc<A: Arbitrary + Debug>() -> QCStrategy<A> {
    QCStrategy::new(qc_gen_size())
}

pub fn from_qc_sized<A: Arbitrary + Debug>(size: usize) -> QCStrategy<A> {
    QCStrategy::new(size)
}
```

**Comment by AltSysrq** [comment]
I would prefer a separate crate. Besides the benefits you mentioned, it also makes it clearer (e.g., for someone looking at it on crates.io) that it's an add-on should one find the use for it rather than a significant-but-optional core feature.

**Comment by Centril** [comment]
A new crate coming your way in a bit =)

**Comment by Centril** [comment]
(Perhaps we should organize stuff into a GitHub organization soon to have repositories in the same place-ish?)

**Comment by Centril** [comment]
The promised crate: https://github.com/Centril/proptest-quickcheck-interop .
Add a note about this to docs perhaps?

**Comment by AltSysrq** [comment]
The crate looks reasonable to me. Only one comment:

https://github.com/Centril/proptest-quickcheck-interop/blob/master/src/lib.rs#L332

Could that `impl` be moved under `mod test` or otherwise made `#[cfg(test)]`? Then it wouldn't escape into the public API.

>  Add a note about this to docs perhaps?

Certainly.

**Comment by Centril** [comment]
Oh! Why didn't I think of that... ;)

Fixed in 1.04.

**Comment by AltSysrq** [comment]
The docs now link to this crate in the paragraph after the introduction.
